docs/tutorials/data_management.md

```diff
@@ -23,18 +23,13 @@ Get a coffee, this might take a while ☕.
 Once installed, run:
 
 ```console
-$ icu4x-datagen --cldr-tag latest --icuexport-tag latest --out my_data_blob.postcard --format blob --all-keys --all-locales
+$ icu4x-datagen --keys all --locales all --format blob --out my_data_blob.postcard
```

Comment:
Possible Issue: `all` is a valid locale, [Allar](https://en.wikipedia.org/wiki/Allar_language) (`none` is fine because it is 4 letters)

Reply:
Yes I thought about this, but it's very unlikely that a client would want to generate *just* Allar, don't you think? If they're generating many single-locale blobs, they should use the API. I think it's fine to say generating only Allar is not supported by the CLI.

Reply:
I don't find that to be a huge concern, personally. Not really a fan of ambiguity in the format

Why not `--locales *`?

Reply:
though that might get shell expanded I guess

Reply:
@macchiati pointed out that these are not actually "all" locales; they are only whatever locales happen to be covered in the current CLDR release. So, `--locales modern` or `--locales basic` could be more precise.

@srl295 suggested that keywords could be expected to be wrapped with some other characters, like `--locales <none>` or `--locales "*"`.

@pedberg-icu agrees that we "should not use anything that could be confused with a potential language code"

Reply:
What's `--locales basic`? PTAL

Reply:
The CLDR coverage levels are `basic`, `moderate`, and `modern`. I think `full` and `basic` correspond to the same set of locales?

Reply:
The CLDR JSON sets are probably modern = {modern} and full = {modern, moderate, basic} then?

Reply:
> The CLDR JSON sets are probably modern = {modern} and full = {modern, moderate, basic} then?

I believe so; @srl295 can you confirm?